system/config/Config.cfc

```diff
@@ -980,6 +980,14 @@ component {
 		};
 
 		settings.rest.authProviders.token = { feature = "restTokenAuth" };
+
+		settings.rest.apis[ "/presideUtils" ]   = {
```

Comment:
Not sure about the naming here. Perhaps `/system/v1` (should include a version number so can make new versions in the future)?